### PR TITLE
Issue 434 422 455

### PR DESCRIPTION
**Pull request by bennahugo**
Fixes a few

### CODE REVIEW

File: cubical/data_handler/ms_tile.py [hunk 1]
```diff
@@ -171,6 +171,7 @@ def load_ddfacet_models(self, uvwco, loaded_models, model_source, cluster, imod,
             ndirs = model_source._nclus
             loaded_models[model_source] = {}
             ddfsim.set_model_provider(model_source)
+            log.info("Predicting targetted directions... This may take some time")
```
Comment (JSKenyon) [inline cubical/data_handler/ms_tile.py:174]: Please amend to targeted.

File: cubical/machines/interval_gain_machine.py [hunk 2]
```diff
@@ -559,12 +559,18 @@ def issue_warning(msg, sort_index=70):
                 if log.verbosity() > 1:
                     msg = "{} directions ({}) flagged {}".format(
                         len(bad_dirs),
-                        ", ".join(["dir {0:s}: {1:.3f}% gains affected".format(
-                            str(d), percflagged[d]) for d in bad_dirs]),
-                        why_flagged
-                    )
+                        ", ".join(["dir {0:s}: {1:s} gains affected".format(
```
Comment (JSKenyon) [inline cubical/machines/interval_gain_machine.py:562]: I would suggest constructing the message incrementally in an if/else ladder. This is very opaque (to me). Disregard if you feel strongly.

**Review by JSKenyon — CHANGES_REQUESTED** [review]
Overall looks good to me - nothing too surprising here. Some minor quibbles you can address at your leisure.

File: cubical/data_handler/ms_tile.py (hunk 1 above)
Comment (bennahugo) [reply cubical/data_handler/ms_tile.py:174]: lol good catch. I always seem to make this mistake

File: cubical/machines/interval_gain_machine.py (hunk 2 above)
Comment (bennahugo) [reply cubical/machines/interval_gain_machine.py:562]: Sorry. As you can tell I like golfing...

File: cubical/data_handler/ms_tile.py (hunk 1 above)
Comment (o-smirnov) [reply cubical/data_handler/ms_tile.py:174]: It's like me and "receipe"... I just cannot train my fingers to do anything else it seems...

**Review by JSKenyon — APPROVED** [review]

**Pull request merged**